PennMobile/src/main/java/com/pennapps/labs/pennmobile/DirectoryFragment.java

```diff
@@ -32,15 +42,15 @@
     private ListView mListView;
     private Context mContext;
     private SearchView searchView;
+    private boolean starOn;
+    private DirectoryAdapter mainAdapter;
+    private Menu mMenu;
+    private int state;
```

Comment:
state of what? Good variable naming is one of the hardest computer science problems